README.md

```diff
@@ -3,7 +3,67 @@ Contribution Guides to the CNCF Ecosystem
 
 Welcome! Are you interested in contributing to one of CNCF hosted projects? This repository should help you contains information and guidelines about contributions to CNCF projects.
 
-CNCF offers multiple ways to start contributing to the CNCF ecosystem, including either foundation-wide and project-wide opportunities.
+CNCF offers multiple ways to start contributing to the CNCF ecosystem, including either foundation-wide and project-wide opportunities. 
+
+Developers
```

Comment:
I'd suggest creating a dedicated document for personas, rather than putting everything to the root README file.